torch/_dynamo/symbolic_convert.py

```diff
@@ -599,6 +602,23 @@ def inline_user_function_return(self, fn, args, kwargs):
             self.restore_graphstate(state)
             raise
 
+    def log_starts_line(self):
+        inline_depth_str = (
+            f"(inline depth: {self.inline_depth}) " if self.inline_depth > 0 else ""
+        )
+        starts_line_log.debug(
+            "TRACE starts_line %s%s %s:%s",
+            inline_depth_str,
+            self.f_code.co_name,
+            self.f_code.co_filename,
+            self.lineno,
+        )
+        line = linecache.getline(self.f_code.co_filename, self.lineno)
+        if line:
+            # strip the newline at the end
+            line = line[:-1]
```

Comment:
If the file is missing trailing newline, line isn't guaranteed to have a newline, right? Maybe rstrip() is better.